### PR TITLE
Add action to deploy demo to Github pages

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: rollup.config.dev.js [hunk 1]
```diff
@@ -7,20 +7,27 @@ import livereload from "rollup-plugin-livereload";
 import sourcemaps from 'rollup-plugin-sourcemaps';
 
 
+const isDev = process.env.DEV === 'true';
```
Comment (bastyen) [inline rollup.config.dev.js]: remove

File: rollup.config.dev.js [hunk 2]
```diff
@@ -7,20 +7,27 @@ import livereload from "rollup-plugin-livereload";
...
+
 export default [
   {
     input: "src/app.js",
     output: {
-      sourcemap: true,
+      sourcemap: isDev,
```
Comment (bastyen) [inline rollup.config.dev.js:16]: replace by !!process.env.DEV

File: rollup.config.dev.js [hunk 3]
```diff
@@ -7,20 +7,27 @@ import livereload from "rollup-plugin-livereload";
...
       dir: "dev",
       format: "es",
     },
     plugins: [
-      scss({
-        output: "dev/css/rando3D.css",
-        watch: 'src/scss',
-        failOnError: true,
-      }),
+      ...isDev ? [
```
Comment (bastyen) [inline rollup.config.dev.js:21]: replace by !!process.env.DEV

File: rollup.config.dev.js [hunk 4]
```diff
@@ -153,9 +160,11 @@ export default [
           return `<!DOCTYPE html><html ${attribute}><head>${meta}<title>${title}</title>${links}${scripts}</head><body>${content}${scripts}</body></html>`;
         },
       }),
-      sourcemaps(),
-      serve({ contentBase: "dev", port: 9000 }),
-      livereload(),
+      ...isDev ? [
```
Comment (bastyen) [inline rollup.config.dev.js:163]: replace by !!process.env.DEV

File: rollup.config.dev.js (hunk 1 above)
Comment (dtrucs) [reply rollup.config.dev.js]: I fixed the variable by `Boolean(process.env.DEV);`

**Pull request merged**